dev/documentation/en/datamodel-keyconcepts.md

```diff
@@ -0,0 +1,100 @@
+---
+book: dev-documentation
+version: 3.4
+title: Entities Key Concepts - Technical Documentation - OJS|OMP|OPS
+---
+
+# Entities: Key Concepts
+
+The overall database schema for the PKP data model consists of well over 100 tables.
+The present page does not introduce the entire data model.
+It merely characterizes a few particularly important entities
+and describes some other things that one might _expect_ to be entities
+but that happen to be represented in some other way.
+
+The page is organized as a loose list of sections in alphabetical order.
+
+When you look for the classes representing these entities,
+some of them will live in PKPlib, others in OJS (etc.),
+and many in both, with an inheritance relationship between them. 
+In the latter case, the superclass in PKPlib usually has a `PKP` name prefix.
+
+## Author
+
+Authors are associated with [Publications](#Publication).
+An Author need not be a [User](#User).
+Except for the email address, an Author's attributes such as
+names and affiliations can be localized.
+
+## Context
+
+Contexts appear _everywhere_ in PKP.
+In OJS, a Context represents a journal.
+Except for Plugins and Users, most things are per context, not shared between contexts.
+
+## Editor
+
+Editors as such are _not_ entities.
+They are Users that get assigned an Editor role in some scope such as
+an entire [Context](#Context) or a Section.
+
+## Publication
+
+Publications represent the various versions of a [Submission](#Submission)
+as they are created during the Submission's lifecycle, e.g. the
+version submitted for initial review, for second review, for publication,
+the version after copyediting, and the final published version.
+Publications would better be called SubmissionVersions (but are not).
+
+Publications represent metadata such as a title and
+are associated with files (via Galley objects)
+and with [Authors](#Author). 
+
+## Reviewer
+
+Reviewers as such are _not_ entities.
+In a general sense, they are [Users](#User) that are permanently earmarked for the Reviewer role.
+In a more specific sense, they are [Users](#User) that are mentioned as Reviewer
+in a [ReviewAssignment](#ReviewAssignment).
+
+## Review
+
+Reviews as such are _not_ entities, because OJS is so flexible regarding what they can look like.
+The shape of a Review is dictated by the ReviewForm set in the ReviewAssignment
+that asked for the Review.
+The content of the Review (according to that ReviewForm) is stored as ReviewFormResponse objects.
```

Comment:
Most people don't use review forms so our description should probably focus on the `ReviewAssignment`. It's best to think of `ReviewAssignment` as the "review" and the `ReviewFormResponse` as an object attached to the assignment.

It's the `ReviewAssignment` where the status is determined (eg - pending, overdue, complete, etc)